cpp/dolfinx/graph/partition.h

```diff
@@ -69,6 +69,25 @@ std::tuple<graph::AdjacencyList<std::int64_t>, std::vector<int>,
 distribute(MPI_Comm comm, const graph::AdjacencyList<std::int64_t>& list,
            const graph::AdjacencyList<std::int32_t>& destinations);
 
+/// @brief Distribute fixed-width nodes to destination ranks.
+///
+/// The global index of each node is assumed to be the local index plus
+/// the offset for this rank.
+///
+/// @param[in] comm MPI Communicator
+/// @param[in] list The list to distribute, as a flattened 2D row major array
```

Comment:
List of what? No need for the doc string to repeat the type, needs to explain what the argument is.